### PR TITLE
Various OSD versions valiation test case added

### PR DESCRIPTION
**Pull request by trepel**
# Description

Added test case about testing different OSD versions for RHOAM fresh install

### CODE REVIEW

File: test-cases/tests/installation/a31-verify-installation-on-all-osd-versions.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+**Trigger the pipeline and analyse the test results**
+
+1. Go to the [RHOAM Addon Flow pipeline](https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/ManagedAPI/job/managed-api-install-addon-flow/) and select "Build with Parameters"
+2. Look for AWS account ID that is free (it doesn't have anything specified in 'Note'). If no account is free, you can use account that is used by nightly pipelines (but don't forget to clean it up for night)
+3. Open the [AWS secrets file from 'vault' repository](https://gitlab.cee.redhat.com/integreatly-qe/vault/-/blob/master/SECRETS.md) locally and look for the AWS credentials for the selected AWS account (aws account id, access key ID and secret access key)
+4. Make sure to fill in `openshiftVersion` properly
+5. Most of the parameters are self-explanatory, many can be left as they are or blank
+
+```
+clusterComputeNodesCount: 6
```
Comment (psturc) [inline test-cases/tests/installation/a31-verify-installation-on-all-osd-versions.md:47]: should we mention here that the cluster should be multiAZ?

**Review by psturc — COMMENTED** [review]
just one comment, otherwise lgtm 👍

File: test-cases/tests/installation/a31-verify-installation-on-all-osd-versions.md (hunk 1 above)
Comment (trepel) [reply test-cases/tests/installation/a31-verify-installation-on-all-osd-versions.md:47]: right, better to have it multiAZ, will update

Comment (trepel) [reply test-cases/tests/installation/a31-verify-installation-on-all-osd-versions.md:47]: @psturc done

**Pull request merged**